docs/Configuration.md

```diff
@@ -162,6 +162,27 @@ resolutions:
 For details of the available reasons, see
 [RuleViolationResolutionReason.kt](../model/src/main/kotlin/config/RuleViolationResolutionReason.kt)
 
+### Curations
+
+#### License Findings
+
+Scan result represent the detected licenses as collection of license findings. A single `LicenseFinding` is represented 
+as tuple: `(license-Id, file-path, start-line, end-line)`. A `LicenseFindingCuration` when applied changes the license-Id
+of any `LicenseFinding` license finding or eliminates the `LicenseFinding` in case the license is set to `NONE`.
+
+Changing similar license findings in all `.cpp` files within the `src` directory from to `Apache-2.0` could be done like
```

Comment:
"...from **`Apache-1.1`** to `Apache-2.0` **can**..."
Maybe rephrase: "As an example, the following curation would replace similar findings of `Apache-1.1` with `Apache-2.0` in all `.cpp` files in the `src` directory:"

Reply:
Done.